src/clib/auto_get_version.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
```

Comment:
This file is automatically generated by the build system. Can you delete this file?